3.evaluate_model/get_LOIO_probabilities_all_featurespaces_no_retraining.ipynb

```diff
@@ -0,0 +1,1258 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #60.            straified_k_folds = StratifiedKFold(n_splits=10, shuffle=False)
</pre><p>Consider removing this code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/phenotypic_profiling_model/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Great point! this must have stuck around and I forgot to delete. Thanks for noting. (I will also delete the GridSearchCV object call)